packages/cli/test/unit/commands/build.test.ts

```diff
@@ -143,6 +143,53 @@ describe('build', () => {
     }
   });
 
+  it('should handle symlinked static files', async () => {
+    const cwd = fixture('static-symlink');
+    const output = join(cwd, '.vercel/output');
+
+    // try to create the symlink, if it fails (e.g. Windows), skip the test
+    try {
+      await fs.unlink(join(cwd, 'foo.html'));
+      await fs.symlink(join(cwd, 'index.html'), join(cwd, 'foo.html'));
+    } catch (e) {
+      return;
```

Comment:
If we know this fails on windows (depending on permissions), let's skip it in windows environments explicitly. We do this elsewhere with:

```
if (process.platform === 'win32') {
  console.log('Skipping test on windows');
  return;
}
```

Reply:
For fun, I wanted to see if GitHub's Windows image supported symlinks and it appears not:

<img width="734" alt="image" src="https://user-images.githubusercontent.com/97262/182714336-b720d14d-ba03-406a-94f9-65245cfb9561.png">